flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/AdaptedRestartPipelinedRegionStrategyNGFailoverTest.java

```diff
@@ -379,10 +385,20 @@ private ExecutionGraph createExecutionGraph(
 			final JobGraph jobGraph,
 			final RestartStrategy restartStrategy) throws Exception {
 
+		final Configuration jmConfig = new Configuration();
+		jmConfig.setBoolean(JobManagerOptions.FORCE_PARTITION_RELEASE_ON_CONSUMPTION, false);
```

Comment:
this should no longer be necessary; `false` is now the default.

Reply:
Yes.